capstone/src/main/webapp/script.js

```diff
@@ -60,12 +60,14 @@ function addRestaurant(restaurant, containerElement) {
  */
 function createRestaurantNameDiv(restaurant) {
   const params = new URLSearchParams();
-  params.append('restaurantKey', restaurant.restaurantKey);
+  params.append(
+      'restaurantKey',
+      restaurant.restaurantKey.slice(9, restaurant.restaurantKey.length - 1));
```

Comment:
Could you add a comment explaining why we have to slice the restaurant key starting at index 9?

Reply:
Oops, that's my bad--it's because when we send Restaurant objects the key comes back as Optional(<key number>) as a String, so the 9 is to cut off Optional and the parenthesis. However, Josh said once we start sending RestaurantHeaders, that will be resolved, so I meant to just keep it as restaurantKey without the slicing--I'll change it back before merging!